packages/template-mrt-reference-app/app/ssr.test.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+const request = require('supertest')
+
+describe('server', () => {
```

Comment:
@olibrook in future PRs should people add tests to this file as they add new functionality to the mrt reference app?

Reply:
Yes, if they feel that's necessary!